src/localization/en.json

```diff
@@ -118,12 +120,30 @@
     "editRequests": "Edit a vacation request",
     "edit": "Edit",
     "cancel": "Cancel",
-    "create": "Request vacation"
+    "create": "Request vacation",
+    "manageRequests": "Manage vacation requests"
   },
   "dataGrid": {
     "noRows": "No Rows"
   },
   "vacationsCard": {
-    "vacations": "Vacations"
+    "vacations": "Vacations",
+    "noUpcomingVacations": "No upcoming vacations",
+    "youHave": "You have",
+    "pendingRequests": "pending vacation requests",
```

Comment:
You should look into formatting strings, this allows us to pass a variable into the string dynamically and would allow you to combine these strings into one. You could find an example of how this is done in oss, if you look at the errorhandling.dialog time or url string strings and see the {0} syntax in the en.json for example

Reply:
I was able to use string formatting like you suggested.